prog/vm/github_runner.rb

```diff
@@ -220,6 +220,8 @@ def setup_info
       # ubicloud/cache package which forked from the official actions/cache package, sends requests to UBICLOUD_CACHE_URL using this token.
       echo "UBICLOUD_RUNTIME_TOKEN=#{vm.runtime_token}
       UBICLOUD_CACHE_URL=#{Config.base_url}/runtime/github/" | sudo tee -a /etc/environment
+      echo '{"ipv6": true, "fixed-cidr-v6": "#{vm.ephemeral_net6.nth(2)}:dc/124", "experimental": false, "dns": ["#{vm.nics.first.private_subnet.net4.nth(2)}", "fd00:0b1c:100d:53::"]}' | sudo tee /etc/docker/daemon.json
+      sudo systemctl restart docker
```

Comment:
Do you have any idea how much this will increase our provisioning times?

Reply:
Runner script allows to run a hook script before the job starts. I'm adding a start hook script for another feature. We might move docker restart to there too.
https://github.com/ubicloud/runner-images/pull/8

Reply:
It adds a "Set up runner" step after "Set up job" step. So it runs before container operations anyway.

https://github.com/enescakir/ubicloud-test-dev/actions/runs/12038401443/job/33563995301